lab3/kursinis.tex

```diff
@@ -377,24 +376,24 @@ \section{Traceability}
     \label{tab:requirements_use_cases}
     \begin{tabular}{|l|*{5}{c|}}
     \hline
-    \backslashbox{Requirement}{Use Case} & \makecell{List \\ Quizzes} & \makecell{Take \\ Quiz} & \makecell{Delete \\ Quiz} & \makecell{Create \\ Quiz} & \makecell{Edit \\ Quiz}\\ \hline
+    \backslashbox{Requirement}{Use Case} & \makecell{List \\ Quizzes} & \makecell{Take \\ Quiz} & \makecell{Delete \\ Quiz} & \makecell{Create \\ Quiz} & \makecell{Quiz Editor \\ Use Cases}\\ \hline
```

Comment:
tas Edit Quiz ir Quiz Editor yra truputi skirting dalykai (as in create ir edit kviečia quiz editor). O jis gavosi, kad tas edit quiz prie quiz editor prisijungė. Turėtų labiau gautis Create, Edit, Quiz editor. Prie Quiz editor kelaius visi set quiz name, upload image ir pan., Create ir Edit tikriausiai užteks individualių use case'ų (FR8 ir FR9), kiti dalykai keliauja į quiz editor.